cub/cub/agent/agent_merge_sort.cuh

```diff
@@ -43,7 +43,9 @@
 #include <cub/util_namespace.cuh>
 #include <cub/util_type.cuh>
 
-#include <thrust/system/cuda/detail/core/util.h>
+#include <thrust/system/cuda/detail/core/load_iterator.h>
+
+#include <cuda/std/__cccl/cuda_capabilities.h>
```

Comment:
This header is part of our global config and should not be included separately
```suggestion
```

Reply:
I included this because I need it for `_CCCL_PDL_GRID_DEPENDENCY_SYNC()`. Which header file should I be including instead?

Reply:
We discussed this offline but the issue is that this header has not been included into the cccl config, so I believe that this is fine for now